lib/routes.js

```diff
@@ -175,5 +177,30 @@ module.exports = ({ cache, config }) => {
     res.end(content)
   }
 
+  exports.overview = async (req, res) => {
+    try {
+      const render = await prepareView()
+
+      const details = {
+        account: config.account,
+        repository: config.repository,
+        date: timeago().format(cache.latest.pub_date),
```

Comment:
Let's use `date-fns` here please 😊 It's faster, slimmer and currently the best option out there.